### ISSUE TITLE
Initial support - AOTAutograd - Test accuracy for TorchBench models

### ISSUE THREAD

**Issue by anijain2305** [pytorch/torchdynamo#86]
List of bugs 
Eager 
- [x] torch.ops.aten.add_ is slow - https://github.com/pytorch/pytorch/issues/74943

Torchscript bugs
- [x] fastNLP_BERT - https://github.com/facebookresearch/torchdynamo/issues/93
- [x] moco - https://github.com/facebookresearch/torchdynamo/issues/94
- [x] slomo - https://github.com/facebookresearch/torchdynamo/issues/95
- [ ] -inf error - https://github.com/facebookresearch/torchdynamo/issues/124
- [x] vision maskrcnn - https://github.com/facebookresearch/torchdynamo/issues/170

TorchDyanamo bugs
- [x] hf_GPT2/speech transformer/hf_t5 - https://github.com/facebookresearch/torchdynamo/issues/85
- [x] tacotron2 - https://github.com/facebookresearch/torchdynamo/issues/82

Torchbench issues
- [x] Most of them tracked at https://github.com/pytorch/benchmark/issues/652

AOTAutograd issues
- [ ] Functionalization tracker - https://github.com/facebookresearch/torchdynamo/issues/88
- [ ] (inplace) pytorch_struct - https://github.com/pytorch/functorch/issues/591
- [ ] (inplace) pytorch_struct second error - https://github.com/pytorch/functorch/issues/514
- [ ] tts_angular - https://github.com/pytorch/functorch/issues/586
- [x] Contiguous tensors - https://github.com/pytorch/functorch/issues/537

NVFuser issues
- [x] Assertion Error 1 - https://github.com/csarofeen/pytorch/issues/1502
- [x] Jit trace fails for training
- [x] Replay error - https://github.com/csarofeen/pytorch/issues/1538
- [x] index error

**Comment by ciciewang** [comment]
need to work on the last 20% of the model.

**Comment by ciciewang** [comment]
will also polish up the 80%.

**Comment by ciciewang** [comment]
resolved one failed issue, one more to go.

**Comment by anijain2305** [comment]
Stack has changed over time. Closing this. Will open new one for errors.
